### DISCUSSION TITLE
Druide Balance

### DISCUSSION THREAD

**Discussion by Rafa28062013** [Hekili/hekili#1017] [Q&A]
Hello,

Why say Hekili i have to use Starfall in Singletarget. I didnt have On this : Use Starfall in Single Target with Stelar Drift. Its out, but why Hekili tell me i have to use it?? 

Here you can see it that i have out.  
![Screenshot (208)_LI](https://user-images.githubusercontent.com/79382964/112355586-18abdc80-8cce-11eb-9352-905eb8ba6e73.jpg)

**Comment by Hekili** [comment]
Because it's now a normal part of the single-target priority when its conditions are met.

```
# Use Starfall with Stellar Drift and no Starlord, when no Starfall is up, you are in Lunar Eclipse or to proc Pulsar when above 90 AsP or adpative swarm has 8+secs remaining or is in flight
actions.st+=/starfall,if=talent.stellar_drift.enabled&!talent.starlord.enabled&buff.starfall.refreshable&(buff.eclipse_lunar.remains>6&eclipse.in_lunar&buff.primordial_arcanic_pulsar.value<250|buff.primordial_arcanic_pulsar.value>=250&astral_power>90|dot.adaptive_swarm_damage.remains>8|action.adaptive_swarm_damage.in_flight)&!cooldown.ca_inc.ready
```
